src/App.tsx

```diff
@@ -15,36 +16,104 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
+enum SortType {
+  ALPHABETIC = 'alphabet',
+  LENGTH = 'length',
+}
+
+const getGoods = (goods: string[], sortType: string, isReversed: boolean) => {
+  const visibleGoods: string[] = [...goods];
+
+  switch (sortType) {
+    case SortType.ALPHABETIC:
+      visibleGoods.sort((a, b) => a.localeCompare(b));
+      break;
+    case SortType.LENGTH:
+      visibleGoods.sort((good1, good2) => {
+        if (good1.length === good2.length) {
+          return good1.localeCompare(good2);
+        }
+
+        return good1.length - good2.length;
+      });
+      break;
+    default:
+      break;
```

Comment:
You don't need to use `break` in the default case. It is already the last case interpreter check